File: contacting-conf-web/src/main/java/nl/ing/api/contacting/conf/domain/model/surveysetting/SurveyDetailsVO.java
package nl.ing.api.contacting.conf.domain.model.surveysetting;

import java.util.List;

public record SurveyDetailsVO(
        SurveySettingVO surveySettingVO,
        List<SurveyPhoneNumberFormatVO> phNumFormats,
        List<SurveyTaskQMappingVO> taskQMappingVO,
        List<SurveyOrgMappingVO> orgMappingVO
) {}

File: contacting-conf-web/src/main/scala/nl/ing/api/contacting/conf/domain/SurveyDetailsVO.scala
package nl.ing.api.contacting.conf.domain

case class SurveyDetailsVO(surveySettingVO: SurveySettingVO, phNumFormats: Seq[SurveyPhoneNumberFormatVO], taskQMappingVO: Seq[SurveyTaskQMappingVO],
                           orgMappingVO: Seq[SurveyOrgMappingVO])
